src/Uno.UI/UI/Xaml/Media/Imaging/Svg/ISvgProvider.cs

```diff
@@ -23,7 +23,7 @@ public interface ISvgProvider
 
 	event EventHandler? SourceLoaded;
 
-	Task<bool> TryLoadSvgDataAsync(byte[] imageData);
+	Task<bool> TryLoadSvgDataAsync(ArraySegment<byte> imageData);
```

Comment:
**This is a breaking change** since `ISvgProvider` is public.

Does someone see a way to implement the optimization without breaking the API? :/

Reply:
I don't think there's a problem breaking this API, Uno is the only consumer. We may also put this API as internal (if we have not documented it)/

Reply:
We haven't, so can be made internal